README.md

```diff
@@ -9,6 +9,8 @@ snippets
 
 [Documentation](http://abouthome-snippets-service.readthedocs.org/)
 
+[![webqa tests](https://img.shields.io/badge/WebQA-Treeherder-lightgrey.svg)](https://treeherder.mozilla.org/#/jobs?repo=snippets-service)
```

Comment:
We don't have any badges at the moment - our main use cases are for projects where a pass/fail is sadly often not clear cut (eg intermittents, coalescing, ...) so would be hard to capture that in a single number.